fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/common/DropColumn.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * (C) Copyright IBM Corp. 2019
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.common;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Arrays;
+import java.util.List;
+
+import com.ibm.fhir.database.utils.api.IDatabaseStatement;
+import com.ibm.fhir.database.utils.api.IDatabaseTranslator;
+
+/**
+ * Drop columns from the schema.table
+ */
+public class DropColumn implements IDatabaseStatement {
```

Comment:
I'm leaning towards NOT dropping columns... 

1 - we always leave a rollback point with the original data, and we can MOVE the data at any point, or re-index. 
2 - it should be relatively transparent. 


Thoughts?

Reply:
I'm torn on it.  I'd prefer not to create it going forward since its never used.  But I don't like the idea of some deployments having it and others not.  Hence the idea to drop it.  I think we could add a step to copy the data from that column into the LOW and HIGH columns if it makes us feel better.
However, I'd prefer to just tell the 2 people that installed version 4.0.0 that they need to be aware of it and just be done.

Reply:
We discussed as a team and decided to NOT drop the old column.
We *do* drop the indices that were using the column though.

Also, we will *not* be adding the column going forward, so users on version 4.0.1 that upgrade to 4.1.0 or higher will have this extra column sitting around.
Also, we do NOT copy this older column's data into the DATE_START or DATE_END columns, so any old resources that had date values indexed should get updated to use the new approach.